### ISSUE TITLE
(Alkmaar) Error doing job: Task exception was never retrieved 

### ISSUE THREAD

**Issue by Sjeff** [pippyn/Home-Assistant-Sensor-Afvalbeheer#212]
Hi,
Since a couple of days i receive this error.
What can i do to resolve this?

```
Traceback (most recent call last):
  File "/config/custom_components/afvalbeheer/sensor.py", line 429, in async_update
    await self.collector.update()
  File "/config/custom_components/afvalbeheer/sensor.py", line 947, in update
    await self.hass.async_add_executor_job(self.__fetch_address)
  File "/usr/local/lib/python3.8/concurrent/futures/thread.py", line 57, in run
    result = self.fn(*self.args, **self.kwargs)
  File "/config/custom_components/afvalbeheer/sensor.py", line 919, in __fetch_address
    response = requests.get(
  File "/usr/local/lib/python3.8/site-packages/requests/models.py", line 900, in json
    return complexjson.loads(self.text, **kwargs)
  File "/usr/local/lib/python3.8/site-packages/simplejson/__init__.py", line 525, in loads
    return _default_decoder.decode(s)
  File "/usr/local/lib/python3.8/site-packages/simplejson/decoder.py", line 370, in decode
    obj, end = self.raw_decode(s)
  File "/usr/local/lib/python3.8/site-packages/simplejson/decoder.py", line 400, in raw_decode
    return self.scan_once(s, idx=_w(s, idx).end())
simplejson.errors.JSONDecodeError: Expecting value: line 1 column 1 (char 0)
```
![image](https://user-images.githubusercontent.com/11430228/110698395-e4c0b980-81ed-11eb-98ff-a0b05afa885d.png)

sensor.yaml:
```
- platform: afvalbeheer
  wastecollector: Alkmaar
  resources:
    - restafval
    - gft
    - papier
    - pmd
  postcode: 1234XX
  streetnumber: xxx
  upcomingsensor: 1
  builtinicons: 0
  disableicons: 0
  dutch: 0
```

**Comment by majbah-ha** [comment]
Hi,

I got exactly the same error for Alkmaar city..
I think the program is not pulling the correct information from the waste collector..
Before that it was working nicely.
I hope the issue will be resolved soon..

**Comment by pippyn** [comment]
The API (inzamelkalender.stadswerk072.nl) has a server issue. It is not possible to request the calendar via the website either. As soon as they fix the issue, the sensor should work again.

**Comment by Sjeff** [comment]
> The API (inzamelkalender.stadswerk072.nl) has a server issue. It is not possible to request the calendar via the website either. As soon as they fix the issue, the sensor should work again.

I can request the calendar for my postalcode via the website. Looks like they have fixed the issue. 
Unfortunately In HA the state is still unknown. can you take another look?

**Comment by Sjeff** [comment]
> Hi,
> 
> I got exactly the same error for Alkmaar city..
> I think the program is not pulling the correct information from the waste collector..
> Before that it was working nicely.
> I hope the issue will be resolved soon..

Last week i switched to afvalinfo to get the correct info from alkmaar https://github.com/heyajohnny/afvalinfo

Had to change a couple of automations but all working again!

**Comment by pippyn** [comment]
Fixed in v4.7.26
